snf-cyclades-app/synnefo/plankton/urls.py

```diff
@@ -73,6 +74,7 @@ def demux_members(request, image_id, member):
 
 urlpatterns = patterns(
     '',
+    (r'^(?:.json)?$', versions_list),
     (r'^v1.0/', include(image_v1_patterns)),
```

Comment:
Also, it seems that the API uses `/v1` and not `/v1.0`, and this is "hardcoded" in the API: http://developer.openstack.org/api-ref/image/v1/index.html

Can you please, add an extra pattern to include v1/
`(r'^v1/', include(image_v1_patterns)),`
and in the version listing field put the `/v1/` version of the URL under the `href` key. We want to move to the more compatible one but retain the old too.